edx_solutions_api_integration/courses/views.py

```diff
@@ -1093,6 +1093,8 @@ def post(self, request, course_id):
         else:
             return Response({}, status=status.HTTP_400_BAD_REQUEST)
 
+        return Response({}, status=status.HTTP_204_NO_CONTENT)
```

Comment:
I think right approach here would to enroll user in the course and send a 201 HTTP status code in response.  Adding these lines  after we have existing_user above should achieve it.
```
CourseEnrollment.enroll(existing_user, course_key)
return Response({}, status=status.HTTP_201_CREATED)
```